docs/developers/documentation/index.md

```diff
@@ -175,41 +203,75 @@ also be caught when previewing the built documentation. If your documentation
 change does not include code, you only need the napari docs dependencies
 installed.
 
-There are two ways you can preview the documentation website, by building
-locally or downloading the GitHub Actions built documentation when you submit
+There are two ways you can preview the documentation website: by building
+locally, or downloading the GitHub Actions built documentation when you submit
 your pull request.
 
-To build the documentation locally, run `make docs` from the root of
-the `napari` repository (assuming you've installed the [docs prerequisites](#prerequisites)).
+To build the documentation locally, run `make docs` from the root of your local
+clone of the `napari/docs` repository (assuming you've installed the
+[docs prerequisites](#prerequisites)).
 
 ```bash
 make docs
 ```
 
-The rendered HTML will be placed in `napari/docs/_build`. Find `index.html` in this folder and drag it
-into a browser to preview the website with your new document.
+````{note}
+The `make docs` command above assumes your have a local clone of the
+[`napari/napari`](https://github.com/napari/napari) repo at the same level as
+the `napari/docs` clone. If that's not the case, you can specify the location of
+the examples gallery folder by executing
+
+```bash
+make docs GALLERY_PATH=<path-to-examples-folder>
+```
+
+The `GALLERY_PATH` option must be given relative to the `docs` folder. If your
+folder structure is 
+
+```
+├── napari-docs
```

Comment:
This should probably be just `docs` right? I mean I also renamed my fork, but for the general case?

Reply:
Well, since in the above instructions I actually cloned it into napari-docs, I've kept the same convention here. I can replace this with just docs everywhere, or make sure folks know they can rename  and use something like <docs repo clone folder>

Reply:
Derp, yeah I see it now. Sorry! I agree, let's leave it to be consistent.
I personally hate the `docs` name, but it is what it is. 
Still as a relative newbie, and putting on my end-user-hat, I think I would appreciate the explicit clarification that the `napari/docs` repo was cloned as `napari-docs` with a clear descriptive phrase. I think potential `docs` fixer-upers may not be git or github gurus so the extra explanation could be worth it.
Just something like:
Note: we've renamed the repo to napari-docs to avoid confusion    ?

Reply:
Yeah, makes sense. I personally have this setup, where I clone it into napari-docs. I have a similar situation with github.com/numpy/archive and github.com/scipy/archive so I've been using this convention already, but I'm happy to make it clearer!